NomaiVR/DebugCheats.cs

```diff
@@ -9,6 +10,10 @@ public class Behaviour : MonoBehaviour
         {
             private void Update()
             {
+#if UNITYEXPLORER
+                UnityExplorer.ExplorerStandalone.CreateInstance();
```

Comment:
Why is this on Update?

Reply:
Ops, I restored this cause I wasn't sure about keeping the change and probably moved it in the wrong place.
Will fix in a bit, maybe I should also move it somewhere that also gets triggered in the main menu...

Reply:
Should be in a better place now, I don't have time to test it though